analytics/pubstack/eventchannel/sender.go

```diff
@@ -0,0 +1,43 @@
+package eventchannel
+
+import (
+	"bytes"
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/analytics/clients"
+	"net/http"
+	"net/url"
+	"path"
+)
+
+type Sender = func(payload []byte)
+
+func NewHttpSender(endpoint string) Sender {
```

Comment:
Please add a httpClient parameter and use it instead of `clients.GetDefaultHttpInstance()`. The client should be passed down from `PubstackModule.configure`.